vogon/urls.py

```diff
@@ -68,8 +71,24 @@
 
 #Error Handlers
 handler403 = 'annotations.exceptions.custom_403_handler'
+schema_view = get_schema_view(
+   openapi.Info(
+      title="Vogon Web API",
+      default_version='v1',
+      description="Vogon's Backend API collection",
+      terms_of_service="https://www.google.com/policies/terms/",
```

Comment:
we don't want to use Google's terms of services. remove if not required, or otherwise put the DigInG url here.